### ISSUE TITLE
Show temperature along icon when on panel

### ISSUE THREAD

**Issue by neoscopio** [rliwoch/plasmoid-wunderground-extended#29]
I use it as a panel widget.
It would be great to have an option to show the temperature along the icon. Right now I have to click or hover.
Only the "default" kde weather widget does this, but it's not working properly.
Thanks

**Comment by Marzal** [comment]
I can see the temperature and the icon
![imagen](https://user-images.githubusercontent.com/2069735/187692564-d813f9ac-1f7e-464a-bd54-a005eaa5a8ad.png)

Is your panel horizontal or vertical?

**Comment by neoscopio** [comment]
vertical.
![image](https://user-images.githubusercontent.com/158853/187881236-4be101fc-a059-4f3e-99f9-8b7457a65f71.png)

**Comment by Marzal** [comment]
Then I think is that, you can try with a horizontal panel and if works is that in a vertical panel is not supported.

**Comment by neoscopio** [comment]
> Then I think is that, you can try with a horizontal panel and if works is that in a vertical panel is not supported.

Sure. Thats why its a feature request. :)

**Comment by Marzal** [comment]
Yeah I mean, that confirming that it works in a horizontal panel and not in a vertical panel could help improve the scope of the feature request, as the functionality is more or less there.

Given the lack of time of the developers I try to give them all the "work" that I can done, give that I can not help coding
